File: java/src/main/java/com/languageweaver/sdk/samples/linguisticoptions/CloudLinguisticOptionsService.java
package com.languageweaver.sdk.samples.linguisticoptions;

import com.languageweaver.sdk.common.cloud.CloudLanguageWeaverClient;
import com.languageweaver.sdk.linguisticoptions.cloud.request.CloudGetLinguisticOptionsRequest;
import com.languageweaver.sdk.linguisticoptions.cloud.result.CloudLinguisticOptionsResult;

public class CloudLinguisticOptionsService {

    public static void main(String[] args) throws Exception {
        try (CloudLanguageWeaverClient lwClient = new CloudLanguageWeaverClient().build()) {
            CloudGetLinguisticOptionsRequest cloudGetLinguisticOptionsRequest = new CloudGetLinguisticOptionsRequest()
                    .setSourceLanguageId("chi")
                    .setTargetLanguageId("eng")
                    .setModel("generic");

            final CloudLinguisticOptionsResult cloudLinguisticOptionsResult = lwClient.getCloudLinguisticOptions(cloudGetLinguisticOptionsRequest);
            // handle result
        }
    }
}

File: dotnet/Samples/LinguisticOptions/CloudLinguisticOptionsService.cs
using LanguageWeaver.Sdk.Common.Cloud;
using LanguageWeaver.Sdk.LinguisticOptions.Cloud.Request;
using LanguageWeaver.Sdk.LinguisticOptions.Cloud.Result;

namespace LanguageWeaver.Sdk.Samples.LinguisticOptions;

public class CloudLinguisticOptionsService
{
    public static void Main()
    {
        using var lwClient = new CloudLanguageWeaverClient().Build();

        CloudGetLinguisticOptionsRequest cloudGetLinguisticOptionsRequest = new CloudGetLinguisticOptionsRequest()
        {
            SourceLanguageId = "chi",
            TargetLanguageId = "eng",
            Model = "generic"
        };
        CloudLinguisticOptionsResult cloudLinguisticOptionsResult =
            lwClient.GetCloudLinguisticOptions(cloudGetLinguisticOptionsRequest);
        // handle result
    }
}